navigator_engine/app.py

```diff
@@ -14,10 +14,12 @@ def create_app(config_object=None):
     app = Flask(__name__)
 
     if not config_object:
-        config_object = os.getenv('CONFIG_OBJECT', 'navigator_engine.config.Config')
+        config_object = os.environ.get(
+            'NAVIGATOR_ENGINE_SETTINGS',
```

Comment:
I suggest keeping more standard env var name for the flask config object `CONFIG_OBJECT`

Reply:
I have... https://flask.palletsprojects.com/en/2.0.x/config/#configuring-from-python-files